cmd/sqlflowserver/main_test.go

```diff
@@ -478,6 +480,80 @@ func TestEnd2EndMaxComputeElasticDL(t *testing.T) {
 	t.Run("CaseTrainElasticDL", CaseTrainElasticDL)
 }
 
+func TestEnd2EndMySQLArgoMode(t *testing.T) {
```

Comment:
Testing MySQL requires starting a MySQL server inside a Pod. Maybe we should use MaxCompute as data source instead.

Reply:
using MySQL can check each PR that comes from forked PR, maybe it can cover more test case.